packages/shared/src/components/profile/ProfileButton.tsx

```diff
@@ -42,7 +44,15 @@ export default function ProfileButton({
             )}
             onClick={wrapHandler(() => onUpdate(!isOpen))}
           >
-            <span className="ml-3 block">{user.reputation ?? 0}</span>
+            <ReputationUserBadge
+              className="ml-3 !typo-callout"
```

Comment:
Nah its ok, perhaps we could rather define types (big/medium) etc and apply styling but it would require a guidelined component.

Lets leave it for now 👍